schemachange/cli.py

```diff
@@ -271,11 +271,24 @@ def authenticate(self):
         warnings.warn(_warn_password, DeprecationWarning)
         snowflake_password = os.getenv("SNOWSQL_PWD")
 
+    # if password
     if snowflake_password:
-      if self.verbose:
-        print(_log_auth_type %  'password' )
-      self.conArgs['password'] = snowflake_password
-      self.conArgs['authenticator'] = 'snowflake'
+        if self.verbose:
+            print(_log_auth_type % 'password')
+        self.conArgs['password'] = snowflake_password
+
+        # if password and okta authenticator
+        if os.getenv("SNOWFLAKE_AUTHENTICATOR").lower()[:8] == 'https://' and os.getenv("SNOWFLAKE_AUTHENTICATOR"):
+          okta = os.getenv("SNOWFLAKE_AUTHENTICATOR")
+          self.conArgs['authenticator'] = okta
+          if self.verbose:
+            print(_log_auth_type % 'Password and Okta')
+            print(_log_okta_ep % okta)
+          else:
+            raise NameError(_err_no_auth_mthd)
```

Comment:
Maybe the spacing is messed up, but it looks like this throws a NameError if verbose if false.